nibabel/gifti/parse_gifti_fast.py

```diff
@@ -108,9 +108,6 @@ def StartElementHandler(self, name, attrs):
             self.img = GiftiImage()
             if 'Version' in attrs:
                 self.img.version = attrs['Version']
-            if 'NumberOfDataArrays' in attrs:
-                self.img.numDA = int(attrs['NumberOfDataArrays'])
-                self.count_da = False
```

Comment:
Yes. I will also add a test for this. In fact, we lack malformed GIFTI file tests; see https://github.com/nipy/nibabel/issues/357